src/main/java/com/minecolonies/coremod/entity/pathfinding/PathJobFindTree.java

```diff
@@ -107,6 +145,38 @@ private boolean isTree(final BlockPos pos)
     {
         if (Tree.checkTree(world, pos, treesToNotCut) && Tree.checkIfInColonyAndNotInBuilding(pos, colony))
         {
+            if (startRestriction != null && endRestriction != null) {
```

Comment:
not sure if this is actually needed with the PathingJob already having restricted coords

Reply:
True, I'll test on that

Reply:
Yeah, he added it because there was a bug in the Path job